### DISCUSSION TITLE
How to use trace search by endpoint name in V8.8.0+

### DISCUSSION THREAD

**Discussion by huailide** [apache/skywalking#8246] [Q&A]
I can use trace search by endpoint name by v8.7.0 ui , but this feature seems to be removed in v8.8.0+.
In V8.7.0 ui I can input some string such as "/markRead" in endpoint name text bar for trace searching.
But in v8.8.0+ I cannot use endpoint name such as "/markRead" to search trace.
When I input some string in the endpoint input bar,nothing happened. The search result has no filter with "/markRead".

v8.8.0+ example img
![880](https://user-images.githubusercontent.com/34831965/144801062-2433866a-e6b2-41e6-88bf-dc9269893310.png)

v8.7.0 example img
![870](https://user-images.githubusercontent.com/34831965/144801136-cbcc8c71-4b2c-44a7-909a-f9a1346f66bd.png)

**Comment by wu-sheng** [comment]
Yes, we removed the endpoint name based search, because it impacts the performance a lot in large scale of trace dataset.
You could click `enter` in the endpoint selector, it would run backend search in that service. Then you could choose the specific one to do the trace query.

**Reply by huailide** [reply]
Thanks for the reply.
When I click endpoint  selector , it didn't show the endpoint that I want to search , e.g. "Mysql". It seems that not all endpoionts shows in the  list. When I input something e.g. "Mysql" it shows nothing.
I was confused . Did I do something not correct?

![880_2](https://user-images.githubusercontent.com/34831965/144805844-efbef6b0-01ae-4115-af1a-7f01428d4f6b.png)
![image](https://user-images.githubusercontent.com/34831965/144806156-d542adf2-e39e-4d77-8af1-c0d8a4dd3e36.png)

**Reply by wu-sheng** [reply]
Sorry, I can tell from this very detail thing. You may need to locate why first.
